docs/examples/milestones/implementation-milestones-3.json

```diff
@@ -0,0 +1,107 @@
+{
+  "uri": "https://standard.open-contracting.org/examples/releases/ocds-213czf-000-00001-05-contract.json",
+  "publishedDate": "2011-01-10T09:30:00Z",
+  "publisher": {
+    "scheme": "GB-COH",
+    "uid": "09506232",
+    "name": "Open Data Services Co-operative Limited",
+    "uri": "https://standard.open-contracting.org/examples/"
+  },
+  "license": "http://opendatacommons.org/licenses/pddl/1.0/",
+  "publicationPolicy": "https://github.com/open-contracting/sample-data/",
+  "version": "1.1",
+  "extensions": [],
+  "releases": [
+    {
+      "ocid": "ocds-213czf-000-00001",
+      "id": "ocds-213czf-000-00001-06-implementation-milestones-3",
+      "date": "2018-12-24T09:30:00Z",
+      "language": "en",
+      "tag": [
+        "implementationUpdate"
+      ],
+      "initiationType": "tender",
+      "parties": [
+        {
+          "identifier": {
+            "scheme": "GB-LAC",
+            "id": "E09000003",
+            "legalName": "London Borough of Barnet"
+          },
+          "name": "London Borough of Barnet",
+          "roles": [
+            "buyer"
+          ],
+          "id": "GB-LAC-E09000003"
+        },
+        {
```

Comment:
Why is the supplier included in this release but not the earlier releases?

Reply:
You are right, I removed also all the not used parties from the other releases.